ansible/roles/package-upgrade/tasks/partials/brew.yml

```diff
@@ -36,10 +36,14 @@
     script: files/install-homebrew.sh
     when: os == "macos11.0" and not armbrew.stat.exists
 
-  - name: Upgrade installed packages
+  # Ignoring deprecation errors in MacOS 10.x
+  - name: Upgrade installed packages 
     become_user: "{{ ansible_user }}"
     homebrew:
       upgrade_all: yes
+    ignore_errors: 
+      - os == "macos10.15" 
+      - os == "macos10.14"
```

Comment:
I think this syntax means "and" and not "or".

Reply:
You are right @richardlau. I fix it in [68c8489](https://github.com/nodejs/build/pull/3130/commits/68c8489c4ec2c7a248c0326682bc2b35214053eb)